src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java

```diff
@@ -713,7 +713,7 @@ public void run() {
                                 SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd_HH.mm.ss");
                                 dateFormat.setTimeZone(TimeZone.getTimeZone("GMT"));
 
-                                final String bundlePrefix = "support";
+                                final String bundlePrefix = BundleNamePrefixProvider.getInstance().getPrefix();
```

Comment:
This needs to be reconciled somehow with the existing SPI, `SupportProvider.getName`.